ctru-rs/examples/software-keyboard.rs

```diff
@@ -21,13 +19,9 @@ fn main() {
     // Custom filter callback to handle the given input.
     // Using this callback it's possible to integrate the applet
     // with custom error messages when the input is incorrect.
-    keyboard.set_filter_callback(Some(Box::new(|str| {
-        // The string is guaranteed to contain valid Unicode text, so we can safely unwrap and use it as a normal `&str`.
-        if str.to_str().unwrap().contains("boo") {
-            return (
-                CallbackResult::Retry,
-                Some(CString::new("Ah, you scared me!").unwrap()),
-            );
+    keyboard.set_filter_callback(Some(Box::new(move |str| {
+        if str.contains("boo") {
```

Comment:
Huh, just realized I'm surprised there's no lint for shadowing builtin `str`, but I guess shadowing in general is not linted for so idk